Dockerfile

```diff
@@ -1,13 +1,39 @@
+FROM python:3.10.1-alpine
+
 ###
-# Install everything we need
+# For a list of pre-defined annotation keys and value types see:
+# https://github.com/opencontainers/image-spec/blob/master/annotations.md
+#
+# Note: Additional labels are added by the build workflow.
 ###
-FROM python:3-alpine as install
-LABEL maintainer="jeremy.frasier@trio.dhs.gov"
-LABEL organization="CISA Cyber Assessments"
-LABEL url="https://github.com/cisagov/saver"
+LABEL org.opencontainers.image.authors="vm-fusion-dev-group@trio.dhs.gov"
+LABEL org.opencontainers.image.vendor="Cybersecurity and Infrastructure Security Agency"
 
-ENV HOME=/home/saver
-ENV USER=saver
+###
+# Unprivileged user setup variables
+###
+# TODO: Change this to 2048.  See cisagov/orchestrator#130 for more
+# details.
+ARG CISA_UID=421
+ARG CISA_GID=${CISA_UID}
+ARG CISA_USER="cisa"
+ENV CISA_GROUP=${CISA_USER}
+ENV CISA_HOME="/home/${CISA_USER}"
+
+###
+# Upgrade the system
+#
+# Note that we use apk --no-cache to avoid writing to a local cache.
+# This results in a smaller final image, at the cost of slightly
+# longer install times.
+###
+RUN apk --update --no-cache --quiet upgrade
+
+###
+# Create unprivileged user
+###
+RUN addgroup --system --gid ${CISA_UID} ${CISA_GROUP} \
+    && adduser --system --uid ${CISA_UID} --ingroup ${CISA_GROUP} ${CISA_USER}
 
 ###
 # Dependencies
```

Comment:
Will you verify that `bash` is needed by this image? The `save_to_db.sh` script does not appear to need it per https://github.com/cisagov/saver/blob/b1b556cfa703e28a9ee09f87bc35c7813d2591c8/src/save_to_db.sh#L1 If it is required will you update this comment block to include a blurb about `bash` like we did in https://github.com/cisagov/scanner/commit/d1d67c011d4824f6d308fd71d92831040c6c174c?

Reply:
You are correct.  `bash` is not required.  Please see commit f8a068a.